### PR TITLE
[Docs] Section update

### PR DESCRIPTION
**Pull request by chandrashritii**
**Description**

- Rearranged the overview page
- Added pages to the Project section

Signed-off-by: Shriti <shritichan@gmail.com>

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/assets/css/main.css [hunk 1]
```diff
@@ -18720,3 +18722,34 @@ footer {
...
+.row {
+    display: flex;
+  }
+  
+.column {
+    flex: 25.00%;
+    padding: 5px;
+  }
+
+.img-thumbnail {
```
Comment (leecalcote) [inline docs/assets/css/main.css:18737]: The book cover image is skewed.
Screen Shot 2020-10-22 at 11 23 23 AM

File: docs/pages/project/resources.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- [Service Mesh Day 2019](https://youtu.be/CFj1O_uyhhs)
+- [DockerCon 2019 Open Source Summit](https://www.docker.com/dockercon/2019-videos?watch=open-source-summit-service-mesh)
+- KubeCon EU 2019 ([video](https://www.youtube.com/watch?v=LxP-yHrKL4M&list=PLYjO73_1efChX9NuRaU7WocTbgrfvCoPE), [deck](https://calcotestudios.com/talks/decks/slides-kubecon-eu-2019-service-meshes-at-what-cost.html))
+- [KubeCon EU 2019 Istio Founders Meetup](https://calcotestudios.com/talks/decks/slides-istio-meetup-kubecon-eu-2019-istio-at-scale-large-and-small.html)
+- [Cloud Native Rejekts EU 2019](https://calcotestudios.com/talks/decks/slides-cloud-native-rejekts-2019-evaluating-service-meshes.html)
+- [Container World 2019](https://calcotestudios.com/talks/decks/slides-container-world-2019-service-meshes-but-at-what-cost.html)
+- Solving the Service Mesh Adopter’s Dilemma ([deck](https://calcotestudios.com/talks/decks/slides-open-source-101-at-home-solving-the-service-mesh-adopters-dilemma.html), [event](https://opensource101.com/sessions/solving-the-service-mesh-adopters-dilemma/),[video](https://www.youtube.com/watch?v=Q1zSWbO0RmI&list=PL3A-A6hPO2IN_HSU0pSfijBboiHggs5mC&index=2&t=0s))
+
```
Comment (leecalcote) [inline docs/pages/project/resources.md:29]: Add something like "Have you given a presentation? Add yours here!" with link to submit docs issue. Be inspired from here - https://linkerd.io/community/

**Comment by leecalcote** [comment]
Back to the drawing board for the administrative-tools.png image.

File: docs/assets/css/main.css [hunk 3]
```diff
@@ -18720,3 +18722,34 @@ footer {
...
+    border: 1px solid #ddd;
```
Comment (leecalcote) [inline docs/assets/css/main.css:18738]: Remove border or if you'd like to make these buttons, reuse the existing button style (or set a new one) on the site.

File: docs/pages/project/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: FAQ
+permalink: project/faq
+---
+
+# FAQ 
+<div class="center" style="color:gray;position:relative;top:-10px;font-size:1.25em;"> Can't find your question here?<br />Post it on the <a href="https://layer5io.slack.com/archives/C01AFD2D547">#support channel</a>
+on the <a hre="http://slack.layer5.io/">Layer5 Slack</a>. </div>
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:9]: keep these on the same line (avoid line break)

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by chandrashritii** [comment]
> 
> 
> Back to the drawing board for the administrative-tools.png image.

I've used a generic icon here. Is there any other image we can use for _service mesh landscape tools_?

**Comment by leecalcote** [comment]
> > Back to the drawing board for the administrative-tools.png image.
> 
> @leecalcote I've used a generic icon here. Is there any other image we can use for _service mesh landscape tools_?



> > Back to the drawing board for the administrative-tools.png image.
> 
> @leecalcote I've used a generic icon here. Is there any other image we can use for _service mesh landscape tools_?

Yes, but icon is out of place with the theme that we have. You might leverage the gear icon from here - 
https://drive.google.com/drive/u/0/folders/1-JzsjwJwCT2yiSRzW0n9qkQq4jDjlq_a

File: docs/pages/project/resources.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/project/resources.md:29]: Created https://github.com/layer5io/meshery/issues/1762 to track this request.

File: docs/assets/css/main.css (hunk 1 above)
Comment (leecalcote) [reply docs/assets/css/main.css:18737]: Created https://github.com/layer5io/meshery/issues/1763 to track this required change.

**Review by leecalcote — APPROVED** [review]
Two issues filed to follow up this PR.

**Pull request merged**